CHANGELOG.md

```diff
@@ -6372,3 +6376,4 @@
 [@FnControlOption]: https://github.com/FnControlOption
 [@ccutrer]: https://github.com/ccutrer
 [@Korri]: https://github.com/Korri
+[@ChrisBr]: https://github.com/ChrisBr
```

Comment:
This line is also not needed as it is auto-generated by the release task. This should be removed.